cmds/tui.c

```diff
@@ -267,6 +294,36 @@ static void print_graph_self(struct field_data *fd)
 	print_time(d);
 }
 
+static void print_graph_self_avg(struct field_data *fd)
+{
+	struct uftrace_graph_node *node = fd->arg;
+	uint64_t d;
+
+	d = node->total_time.avg - node->child_time.avg;
+
+	print_time(d);
+}
+
+static void print_graph_self_min(struct field_data *fd)
+{
+	struct uftrace_graph_node *node = fd->arg;
+	uint64_t d;
+
+	d = node->total_time.min - node->child_time.min;
```

Comment:
This won't work correctly.  The min (and max) of self time should be calculalted each time it adds a graph node like report does.